roles/lib_openshift/library/oc_obj.py

```diff
@@ -1531,14 +1533,11 @@ def run_ansible(params, check_mode=False):
         if state == 'list':
             return {'changed': False, 'results': api_rval, 'state': 'list'}
 
-        if not params['name']:
```

Comment:
I followed the logic found in `_get()`. My guess was that since `oc get` (and similarly `oc delete`) would produce an error if both name and selector were missing that error gets reported back by `return self.openshift_cmd(cmd)`.